v2/objective-c/.gitignore

```diff
@@ -8,10 +12,17 @@ build/
 *.perspectivev3
 !default.perspectivev3
 xcuserdata
+
+# Other
 *.xccheckout
 *.moved-aside
-DerivedData
+*.xcuserstate
+*.xcscmblueprint
+
+# Obj-C/Swift specific
 *.hmap
 *.ipa
-*.xcuserstate
 
+# CocoaPods
```

Comment:
@mittonk, readme is updated now! If it looks good to you, it should be ready to go.